ro-base.obo

```diff
@@ -5861,11 +5910,132 @@ is_a: RO:0002449 ! directly negatively regulates activity of
 
+[Typedef]
+id: RO:0018030
+name: chemical relationship
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0003-4423-4370
+property_value: IAO:0000232 "Do not use this relation directly. It is intended as a grouping for a diverse set of relations, in which the subject or object is a chemical." xsd:string
+
+[Typedef]
+id: RO:0018031
+name: is direct conjugate base of
+def: "A is a direct conjugate base of B if and only if A is chemical entity that is a Brønsted–Lowry Base (i.e., can receive a proton) and by receiving a particular proton transforms it into B." []
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0003-4423-4370
+property_value: IAO:0000112 "pyruvate anion (CHEBI:15361) is the conjugate base of the neutral pyruvic acid (CHEBI:32816)" xsd:string
+property_value: RO:0002575 RO:0018033
+property_value: seeAlso https://github.com/oborel/obo-relations/issues/643
+is_a: RO:0018033 ! is deprotonated form of
+inverse_of: RO:0018032 ! is direct conjugate acid of
+
+[Typedef]
+id: RO:0018032
+name: is direct conjugate acid of
+def: "A is a direct conjugate acid of B if and only if A is chemical entity that is a Brønsted–Lowry Acid (i.e., can give up a proton) and by removing a particular proton transforms it into B." []
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0003-4423-4370
+property_value: IAO:0000112 "neutral pyruvic acid (CHEBI:32816) is the conjugate acid of the pyruvate anion (CHEBI:15361)" xsd:string
+property_value: RO:0002575 RO:0018034
+property_value: seeAlso https://github.com/oborel/obo-relations/issues/643
+is_a: RO:0018034 ! is protonated form of
+
+[Typedef]
+id: RO:0018033
+name: is deprotonated form of
+def: "A is a deprotonated form of B if and only if A is chemical entity that is a Brønsted–Lowry Base (i.e., can receive a proton) and by adding some nonzero number of protons transforms it into B.\n\nThis is a transitive relationship and follows this design pattern: https://oborel.github.io/obo-relations/direct-and-indirect-relations." []
+xref: is_conjugate_base_of
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0003-4423-4370
+property_value: IAO:0000112 "(E)-cinnamoyl-CoA(4-) (CHEBI:57252) is a deprotonated form (E)-cinnamoyl-CoA (CHEBI:10956), which involves removing four protons." xsd:string
+property_value: seeAlso https://github.com/oborel/obo-relations/issues/643
+is_transitive: true
+is_a: RO:0018030 ! chemical relationship
+inverse_of: RO:0018034 ! is protonated form of
+
+[Typedef]
+id: RO:0018034
+name: is protonated form of
+def: "A is a protonated form of B if and only if A is chemical entity that is a Brønsted–Lowry Acid (i.e., can give up a proton) and by removing some nonzero number of protons transforms it into B.\n\nThis is a transitive relationship and follows this design pattern: https://oborel.github.io/obo-relations/direct-and-indirect-relations." []
+xref: is_conjugate_acid_of
```

Comment:
?

Reply:
these should be xrefs to chebi hash IRIs, maybe a bug in the OWL->OBO conversion doesn't consider that hash IRIs can come from a specific namespace

Reply:
so in OBO, should this be written as a CURIE like `obo:chebi#is_conjugate_acid_of`?

Reply:
Yes, this will be a bug which will take 100 years to be fixed, so better use seeAlso for this, or skos:exactMatch!

Reply:
Done in https://github.com/oborel/obo-relations/pull/729 in case you want to include that in this release